README.md

```diff
@@ -208,11 +208,39 @@ $googleAdsClient = (new GoogleAdsClientBuilder())
 Once you have an instance of `GoogleAdsClient`, you can obtain a service client
 for a particular service using one of the `get...ServiceClient()` methods.
 
+## Transport
```

Comment:
Provided that this is a nice-to-have feature, we should put this section in [DevSite](https://developers.google.com/google-ads/api/docs/client-libs/php/) instead.
Then, you can just briefly mention above (around the **Requirement** section) that the property file can be configured to specify transport.

Reply:
Agreed, the documentation is transferred to a new guide: https://developers.google.com/google-ads/api/docs/client-libs/php/transport.